docker-compose.yml

```diff
@@ -201,6 +201,7 @@ services:
     image: cernopendata/nginx
     ports:
       - "80:80"
+      - "443:443"
```

Comment:
Speaking of SSL, there is quite a difference in the number of warnings seen in the web browser console. Shall we address them now or later?

HTTP:

![http](https://user-images.githubusercontent.com/517546/104163608-fad11a00-53f6-11eb-827f-1722b54fd968.png)

HTTPS:

![https](https://user-images.githubusercontent.com/517546/104163628-01f82800-53f7-11eb-8f4a-a93818d82d15.png)

Reply:
They seem very localised, `csrftoken` and static assets. Perhaps we can fix it in a separate issue as it is "independent" from this task (altough it might be related with the Flask-Talisman config, there are chances we would hit this once we deploy and serve the upgraded Invenio site through HTTPS). Also it would allow us to move faster merging what we have today in the pipeline, what do you think?

Reply:
Sure, we can ticketise this for later... Please update the other two things and we can merge.